### ISSUE TITLE
这个数据可以用来训练模型吗

### ISSUE THREAD

**Issue by pengpengtao** [rany2/edge-tts#278]
我在训练过程中，用这个数据训练不了，而用其他开源数据却没有问题，数据是加了水印吗

**Comment by pengpengtao** [comment]
> 这是完全可能的。不幸的是，我无法得出结论性回答，但这是有道理的。出于好奇，您用什么来训练模型？

![image](https://github.com/user-attachments/assets/0d016c6d-9f88-4d1c-a6a6-eb3226da8c57)
I am using VITS2 for training, but during the training process, the predicted mel spectrogram becomes NaN, which causes the training to terminate. I suspect there might be an issue with the generated audio by the model, because training with other datasets does not present this problem.

**Comment by rany2** [comment]
I'm not sure if you've done that, but i think you do need to convert all the mp3 from edge-tts to WAV.

**Comment by pengpengtao** [comment]
> 所有者

Yes, I have already converted the MP3 files to WAV format. The issue is not that it goes wrong at the first epoch, but rather it starts to fail after a certain number of steps during training. The mel spectrogram loss spikes, becoming very high, and then NaN values appear.

**Comment by pengpengtao** [comment]
> It's probably an intentional thing. I don't know how it's achieved technically but it's interesting.


So is this data unusable, and cannot be used as generated data to train other models?

**Comment by rany2** [comment]
Unfortunately I don't know but it does appear this way from your info.
